### PR TITLE
gui-apps/waybar: add 0.9.17, drop 0.9.12

### PR DESCRIPTION
**Pull request by MrRoy**
Closes: https://bugs.gentoo.org/852302
Signed-off-by: Julien Roy <julien@jroy.ca>

### CODE REVIEW

**Comment by MrRoy** [comment]
Please do not merge this yet; there is an [issue](https://github.com/Alexays/Waybar/issues/1852) with 0.9.17. Upstream is working on a fix, which I will include to this PR when it is ready.

**Comment by MrRoy** [comment]
I have added a patch for 0.9.17, so it could be merged now if everything else is ok.

**Comment by MrRoy** [comment]
Since it is not merged yet, I included another patch with an improved fix for the previously mentioned bug.

File: gui-apps/waybar/files/waybar-0.9.17-wireplumber.patch [hunk 1]
```diff
@@ -0,0 +1,371 @@
+From 4e8ccf36b54cacf5281726d23ea14312a133f977 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline gui-apps/waybar/files/waybar-0.9.17-wireplumber.patch:1]: Please include a link to the relevant upstream commit and bug(s).

**Pull request closed without merging**